whois-nrtm4/src/main/java/net/ripe/db/nrtm4/domain/NrtmSourceModel.java

```diff
@@ -0,0 +1,5 @@
+package net.ripe.db.nrtm4.domain;
+
+public record NrtmSourceModel(long id, NrtmSource source) {
```

Comment:
What do you think of this? 
Use NrtmSource itself and add id there, we can put jsonIgnore for it. Source will always have id so I am not sure why we have three domain classes for sources.

Reply:
But properties are same for every machine as we copy from single location. Also we use jmx for creating initial source in tables so it can never happen that it will be created by two instances at same time? and we should have unique constraint on name in table schema?